c_glib/arrow-glib/table-builder.cpp

```diff
@@ -241,6 +241,9 @@ garrow_record_batch_builder_get_n_fields(GArrowRecordBatchBuilder *builder)
  *   the `i`-th field on success, %NULL on out of index.
  *
  * Since: 0.8.0
+ *
+ * Deprecated: 0.13.0:
+ *   Use garrow_record_batch_builder_get_column_builder() instead.
```

Comment:
Could you call  `garrow_record_batch_builder_get_column_builder()` in this function?

Reply:
Thank you for pointing out. I've fixed it.